### DISCUSSION TITLE
Demo Login

### DISCUSSION THREAD

**Discussion by MentalGear** [wagtail/wagtail#9676] [General]
The page on the website linking to the demo. It doesn't show which credentials to use to login.

**Comment by gasman** [comment]
Thanks for the report @MentalGear! Have now opened this as an issue on https://github.com/wagtail/wagtail.org/issues/341.

**Comment by ArSenic04** [comment]
Can I contribute to it?

**Reply by RealOrangeOne** [reply]
The codebase for the old demo, what we're now using as our internal demo site and what others use as their first intro to Wagtail is at https://github.com/wagtail/bakerydemo.

**Comment by gasman** [comment]
@ArSenic04 No, not any more. As per the issue linked above:

> The demo link has now been removed. The old demo site was only meant to be used as an internal testing tool when it was first setup (hence "nightly"), and has now gone back to being just that.
